use-cases/using-SRIOV-and-kata.md

```diff
@@ -316,3 +317,33 @@ The following example launches a Kata Containers container using SR-IOV:
    ```
    sriov-2:~$ sudo docker run --runtime=kata-runtime --net=vfnet --cap-add SYS_ADMIN --ip=192.168.0.11 -it mcastelino/iperf iperf3 -c 192.168.0.10 bash -c "mount -t ramfs -o size=20M ramfs /tmp; iperf3 -c 192.168.0.10"
    ```
+## Example 2: Launch a Kata container using SR-IOV without SR-IOV Plugin
+
+The second example launches a Kata container using SR-IOV, however doesn't needs an additional plugin.
+A self-built guest kernel, which includes the appropriate NIC driver kernel module, might be necessary.
+
+1. Get PCI address of VF in form "0000:xx:xx.x":
+   ```sh
+   $ PCI=<PCI address of VF>
+   ```
+2. Get IOMMU group of VF (should be a number):
+   ```sh
+   $ GROUP=$(readlink -e /sys/bus/pci/devices/$PCI/iommu_group | sed "s/.*\///")
+   ```
+
+3. Unbind VF from current network driver (driver name might be different):
+   ```sh
+   $ echo "$PCI" | sudo tee /sys/bus/pci/drivers/i40e/unbind
+   ```
+
+4. Bind VF to `vfio-pci` driver:
+   ```sh
+   $ echo "vfio-pci" | sudo tee /sys/bus/pci/devices/$PCI/driver_override
+   $ echo "$PCI" | sudo tee /sys/bus/pci/drivers/vfio-pci/bind
+   ```
+
+5. Hand `vfio-pci` device handle to container:
+   ```sh
+   $ sudo docker run --runtime=kata-runtime --cap-add NET_ADMIN --cap-add SYS_ADMIN -device /dev/vfio/$GROUP -v /dev:/dev ...
```

Comment:
Would it be helpful to write a note about why each capabilities is required and why entire /dev is to be mounted instead of /dev/vfio/vfio (apart from /dev/vfio/$GROUP)?

Reply:
@HolyShitMan - I think this is the last comment to address before we can land this.